test/verify/check-packagekit

```diff
@@ -152,6 +152,9 @@ class TestUpdates(NoSubManCase):
 
         self.login_and_go("/updates")
 
+        # on supported systems we show a spinner
+        b.wait_visible("#kpatch-settings svg.pf-v6-c-spinner")
```

Comment:
Isn't this racy? The spinner might have gone away by the time we come to check for it here, no?